inst/templates/pipe.R

```diff
@@ -8,4 +8,7 @@
 #' @export
 #' @importFrom magrittr %>%
 #' @usage lhs \%>\% rhs
+#' @param lhs A value or the magrittr placeholder.
+#' @param rhs A function call using the magrittr semantics.
+#' @returns the result of calling `rhs(lhs)`.
```

Comment:
Morally, it feels like this should be:

```
#' @inheritParams magrittr::`%>%`
#' @inherit magrittr::`%>%` return
```

and yet I believe it can't really be so, because here we are just documenting `NULL`, so at `document()`-time we see:

```
Warning message:
Topic '%>%': no parameters to inherit with @inheritParams 
```

And no return value is documented because `` magrittr::`%>%` `` itself does not technically document a return value (https://magrittr.tidyverse.org/reference/pipe.html).

So maybe this is the best plan? @Hadley do you have an opinion?

Reply:
Yeah, I think this approach is fine. I don't think many people will actually read this documentation, so this is just about satisfying CRAN.